app/javascript/chat/components/ChannelImage.jsx

```diff
@@ -2,9 +2,32 @@ import { h } from 'preact';
 import PropTypes from 'prop-types';
 import { defaultChannelPropTypes } from '../../common-prop-types/channel-list-prop-type';
 
-export function ChannelImage(props) {
-  const { channel, newMessagesIndicator, discoverableChannel } = props;
+/**
+ * Returns an image to help users identify chat channels
+ *
+ * @param {object} channel - Contains information about the channel this image represents
+ * @param {string} newMessagesIndicator - Used to construct a CSS classname
+ * @param {boolean} discoverableChannel - Used to determine which CSS class should be applied
+ *
+ * @example
+ * <ChannelImage
+     channel={{channel_name: "Example Channel",
+               channel_type: "direct",
+               channel_modified_slug: "someslug-f7ff2c5a6a",
+               id: 1,
+               chat_channel_id: 20,
+               status: "active",
+               channel_image: "/some/path/to/image"}}
+     newMessagesIndicator="old"
+     discoverableChannel=false
```

Comment:
(nitpick / not blocking)
```suggestion
     discoverableChannel={false}
```